mappings/net/minecraft/block/RedstoneWireBlock.mapping

```diff
@@ -1,15 +1,23 @@
 CLASS net/minecraft/unmapped/C_hbdlqish net/minecraft/block/RedstoneWireBlock
+	FIELD f_arszcewt N I
 	FIELD f_awrxoajc DOT_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
 	FIELD f_bcvuxzmx POWER Lnet/minecraft/unmapped/C_vltzvhxi;
 	FIELD f_fyjjmxtv WIRE_CONNECTION_SOUTH Lnet/minecraft/unmapped/C_cgckxfsw;
 	FIELD f_isurntyc WIRE_CONNECTION_NORTH Lnet/minecraft/unmapped/C_cgckxfsw;
+	FIELD f_jbpgiwve S I
 	FIELD f_jjwmffoj DIRECTION_TO_WIRE_CONNECTION_PROPERTY Ljava/util/Map;
 	FIELD f_laswtpnf COLORS [Lnet/minecraft/unmapped/C_vgpupfxx;
 	FIELD f_lhkdefum WIRE_CONNECTION_WEST Lnet/minecraft/unmapped/C_cgckxfsw;
+	FIELD f_lsmbxxxx SHAPES_FLOOR Ljava/util/Map;
+	FIELD f_nakuehnf H I
```

Comment:
What's this one?

Reply:
Not overly sure

Reply:
It might be the height of the shapes in `SHAPES_FLOOR` and `SHAPES_UP`

Reply:
ah true

Reply:
Eventually gets turned into `BitSetVoxelSet::maxY`, so yes, this is height